src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/version/fixtures.js

```diff
@@ -0,0 +1,81 @@
+export const CPSVersionBlock = {
+  id: '63501296',
+  subType: 'primary',
+  format: 'video',
+  externalId: 'journalism_world_service_stream_01',
+  duration: 'PT0S',
+  caption: 'live stream from event',
+  image: {
+    id: '63501296',
+    subType: 'thumbnail',
+    href: 'http://b.files.bbci.co.uk/15E0C/test/_63721698_p01kx435.jpg',
+    path: '/cpsdevpb/15E0C/test/_63721698_p01kx435.jpg',
+    height: 360,
+    width: 640,
+    altText: 'connectionAltText',
+    copyrightHolder: 'BBC',
+    type: 'image',
+  },
+  embedding: false,
+  available: true,
+  live: true,
+  type: 'version',
+};
+
+export const optimoVersionBlock = {
```

Comment:
I would of expected us to convert a CPS `version` block into a Optimo `video` block, that way we can share the logic of the `video` block using the `articleMediaPlayer`?

Reply:
The CpsAssetMediaPlayer doesn't seem to care about the `type` of the block.  That is only relevant in the `componentsToRender` object, that maps types to components

We can pass the "version" block to the "CpsAssetMediaPlayer" block and it will work fine

We could also just change the "version" blocks to "media" blocks, but it seems there is some value to retaining the distinction - we will have different behavior eventually, and for now it does stop the live streams from rendering until we actually add them to the `componentsToRender` map